src/map/guild.cpp

```diff
@@ -874,7 +874,7 @@ int guild_recv_info(const struct mmo_guild &sg) {
 		sd = g->guild.member[i].sd;
 		if( sd==nullptr )
 			continue;
-		sd->guild = g;
+		clif_name_area(&sd->bl); // [LuzZza]
```

Comment:
```suggestion
		if( sd->guild == nullptr ){
			sd->guild = g;
			clif_name_area(&sd->bl);
		}else{
			sd->guild = g;
		}
```
Should it not be like that?

Reply:
In regards of fixing the mentioned bug my approach worked fine (I placed the guild assignment next to assigning the member sd, considering the first loop as "assignment" loop), however the suggested condition ensures the name update is not sent when not necessary, e.g. when guild data is received after skill level change.